### ISSUE TITLE
Update E2E testing

### ISSUE THREAD

**Issue by elisabettai** [ITISFoundation/osparc-issues#1208]
**Describe the user role**
As coders, it would be nice to update the e2e in a similar way as it is done in the TI Repo

**Describe the goal**
Have input and outputfiles if needed in the form of .yaml (.toml) files where we write the inputs and outputs we have. Then the input is fed into the code sections and the output of the algo is compared to the output file.

**Describe the benefit**
Making the testing easier.

**Comment by Konohana0608** [comment]
@sanderegg @odeimaiz  Do you guys think this case describes something that can realistically be done? From what I saw, updating E2E tests is much more involved than just changing inputs and outputs. Especially with GUI changes the main code of the E2E has to be touched.

**Comment by sanderegg** [comment]
@Konohana0608 
I think I might have inadvertently put that in the list of cases. this is from 2023. Not sure what this exactly means. we should probably just close it? otherwise I would recommend modifying its description

**Comment by Konohana0608** [comment]
@sanderegg Yes, I am also not really sure how exactly Elisabetta and I meant that ^^. Back then, I didn't properly understand how e2e tests were set up with our infrastructure. Reading it now, it doesn't make a lot of sense to me either. I will close it
